admin/src/pages/UserSearch.spec.js

```diff
@@ -178,8 +178,8 @@ describe('UserSearch', () => {
                 searchText: '',
                 currentPage: 1,
                 pageSize: 25,
-                notActivated: null,
-                isDeleted: null,
+                filterByActivated: null,
+                filterByDeleted: null,
```

Comment:
```suggestion
                filterByActivated: false,
                filterByDeleted: false,
```

Reply:
I like that we can see explizit that we switsch between `null` and `false` or `true`.
We start with `null` here in both cases.